period/flag.go

```diff
@@ -0,0 +1,14 @@
+package period
+
+func (period *Period) Set(p string) error {
```

Comment:
Please add a comment to describe the purpose, e.g.

```
// Set enables use of Period by the flag API.
```
(or similar words)

Reply:
Done!